example/dash_analytics_example.dart

```diff
@@ -4,13 +4,12 @@ import 'package:file/file.dart';
 import 'package:file/local.dart';
 
 import 'package:dash_analytics/dash_analytics.dart';
+import 'package:dash_analytics/src/enums.dart';
```

Comment:
package consumers generally shouldn't import files under `lib/src` (there's an analyzer lint for this, but apparently it's not part of package:lints?) (note, it's ok for tests to directly import any file). Instead, can you make `package:dash_analytics/dash_analytics.dart` export everything that users would need to directly reference?

Reply:
This is an example of dart file at the top-level lib directory that exists to control what from lib/src will be exported to end users: https://github.com/flutter/flutter/blob/master/packages/flutter/lib/foundation.dart

Reply:
Oh whoops, I completely forgot to replace that – the auto import is too good 😄